scss/layout/_panel-grid.scss

```diff
@@ -7,7 +7,7 @@
 $block: #{$fd-namespace}-panel-grid;
 .#{$block} {
   //VARS
-
+  @include fd-reset();
```

Comment:
nit: this is slightly awkward underneath `// VARS` :)

it might be better above the properties on line 14?

Reply:
😅 ha! yes